src/content/self-hosted/install/deployment.mdx

```diff
@@ -270,6 +270,8 @@ You can also use the `okteto-cloud-secret` secret to store the following values
 - `GITHUB_CLIENTSECRET`: use this instead of `.Values.auth.github.clientSecret` in your helm configuration file.
 - `BITBUCKET_CLIENTSECRET`: use this instead of `.Values.auth.bitbucket.clientSecret` in your helm configuration file.
 - `OPENID_CLIENTSECRET`: use this instead of `.Values.auth.openid.clientSecret` in your helm configuration file.
+- `GITHUB_INTEGRATION_CLIENTSECRET`: use this instead of `.Values.github.clientSecret` in your helm configuration file.
```

Comment:
Can we add a reference from here https://www.okteto.com/docs/self-hosted/administration/github/#adding-the-github-app-configuration-to-okteto?
Otherwise this is very hard to discover by our users

Reply:
what part is hard to discover?

Reply:
that these values can be defined in a secret instead of putting them in the okteto helm values file. if you are creating the github app, we should indicate in that doc that we can configure these values in a secret

Reply:
I added it in https://deploy-preview-249--okteto-docs.netlify.app/docs/self-hosted/administration/github/#adding-the-github-app-configuration-to-okteto, check it out see if that's what you meant?

Reply:
Right, that's what I meant :-)